docs/how-tos/deploy-ml.md

```diff
@@ -137,34 +289,39 @@ Click **Train model** and follow the prompts.
 You can choose between
 
 - training a new model or updating a model
-- You can use a **Built-in training** script or a [custom training script](/services/ml/training-scripts/).
+- You can use a **Built-in training** script or a [training script](/services/ml/training-scripts/).
```

Comment:
This is confusing. A built-in training script is still a training script, so putting it this way makes it not either-or.

Also I don't think "built-in training" should be bold.

Should it be this?

```suggestion
You can train or update using a [built-in training script](/services/ml/training-scripts/) or your own [custom training script](/how-tos/create-custom-training-scripts/).
```

Reply:
```suggestion
You can train or update using **Built-in training** or using a [training script](/services/ml/training-scripts/) from the Viam Registry.
```